dcos_launch/config.py

```diff
@@ -368,7 +380,6 @@ def _validate_genconf_scripts(genconf_dir, dcos_config):
         'type': 'dict',
         'required': True,
         'allow_unknown': True,
-        'default_setter': lambda doc: yaml.load(util.read_file(os.path.join(doc['genconf_dir'], 'config.yaml'))),
```

Comment:
What's the reason for removing this?

Reply:
Its moved up, I'm not that familiar with `cerberus` but I believe that since it `ONPREM_DEPLOY_COMMON_SCHEMA` is an update to `COMMON_SCHEMA` it doesn't make sense to have it defined and then just overwritten. I suspect the rest of the schema can be removed as well, but I didnt' try that yet.